python/ngen_cal/src/ngen/cal/model.py

```diff
@@ -137,7 +137,7 @@ def param_log_file(self) -> Path:
         """
             The path to the best parameter log file
         """
-        if id is not None:
+        if self.id is not None:
```

Comment:
Should the logic here be reversed?

```suggestion
        if self.id is not None:
            prefix = f"{self.id}_"
        else:
            prefix = ""
```

Reply:
yes, rebased into 8f0fb8778eeb818345734343455672c753bcba74